src/app/datagrid/hide-show-columns/hide-show.html

```diff
@@ -12,49 +12,49 @@
 <button class="btn btn-outline-primary" (click)="toggleDate()">
     Toggle Date
 </button>
-<clr-datagrid>
-    <clr-dg-column *ngIf="showId">
-        <ng-container *clrDgHideableColumn>
-            User ID
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <!--Name-->
-        <ng-container *clrDgHideableColumn>
-            Name
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column *ngIf="showDate">
-        <ng-container *clrDgHideableColumn>
-            Creation date
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <ng-container *clrDgHideableColumn>
-            Pokemon
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <ng-container *clrDgHideableColumn>
-            Favorite color
-        </ng-container>
-    </clr-dg-column>
+    <clr-datagrid>
```

Comment:
Not sure if this diff is because of a formatting change but normally formatting changes should be done in a separate PR because it creates a longer diff not related to the actual change and just in case we revert this commit, the formatting changes are lost too. same for the other formatting changes in this file.